source/dofs/dof_handler_policy.cc

```diff
@@ -3513,6 +3575,12 @@ namespace internal
         const unsigned int
         n_cpus = Utilities::MPI::n_mpi_processes (triangulation->get_communicator());
 
+
+        // at the beginning, make sure every processor knows the
+        // active_fe_indices on both its own cells and all ghost cells
+        communicate_active_fe_indices (*this->dof_handler);
```

Comment:
Not yet, but I can at least put the call in place already, given that `p::d::Tria` isn't supported yet anyway. I'm adding the call because it will need to be added at one point anyway.